app/components/windows/SourcesShowcase.vue

```diff
@@ -11,7 +11,8 @@
       v-if="inspectedSource === 'image_source'"
       @clickAdd="selectSource('image_source')"
       name="Image"
-      description="Add images to your scene.">
+      description="Add images to your scene."
+      key="1">
```

Comment:
Why are we adding a key to these elements?  Normally you would only use a key when you have a `v-for` loop of dynamically generated elements, and you want to distinguish them.

Reply:
I changed the media from gifs to mp4s which caused a bug where the path of the source would change but the actual video element wouldn't reload so the wrong video would play. This can happen when you have multiple on the same component on a page so I had to add a key attribute to fix this.

Reply:
Ideally I will refactor this whole page in the future to be more dynamic and use a JavaScript object to fill the data

Reply:
Ah I see.  That's fine.  I agree eventually we should refactor this to render off of a JSON file or something.